server/src/bots/telegram/botResponse/countryResponse.ts

```diff
@@ -22,12 +25,19 @@ export const showCountries = (bot, message) => {
 
 const getCountryFromMessage = (userTextCode): string => userTextCode.slice(userTextCode.indexOf(' ')).trim();
 
-export const showCountry = async (bot, message): Promise<void> => {
-    const requestedCountry: UserPresentationalCountryNameString = adaptCountryToSystemRepresentation(getCountryFromMessage(message.text));
+export const showCountryByName = async (bot, message): Promise<void> => {
 
+    return showCountry(bot, message, adaptCountryToSystemRepresentation(getCountryFromMessage(message.text)));
+};
+
+export const showCountryByFlag = async (bot, message): Promise<void> => {
+    return showCountry(bot, message, name(message.text));
```

Comment:
did you check either it case sensative (it should not be, e.g. it should work for UKRAINE, Ukraine, ukraine, uKraine etc)

Reply:
![image](https://user-images.githubusercontent.com/25850290/77904704-ac9ac200-728d-11ea-8558-cfda131fc80e.png)